### DISCUSSION TITLE
I miss Fief already :(

### DISCUSSION THREAD

**Discussion by kschniedergers** [fief-dev/fief#324] [General]
With the discontinuation of the cloud offer, I unfortunately had to switch to auth0 since self hosting isn't an option I'm interested in, and I just want to say that using auth0 has been so much worse and so much more of a headache than fief ever was.

I miss it so much already, if it ever comes back, I will be first in line to use it again.

**Comment by frankie567** [comment]
Hi @kschniedergers!

Sorry to hear that ☺️ Unfortunately, I have to make difficult choices sometimes to keep the project on track. With that said, even though I'll discontinue the cloud offer, we'll provide some options so you can self-host really easily on cloud platforms with very minimal maintenance needed (like [DigitalOcean Marketplace](https://marketplace.digitalocean.com/) or [Plural.sh](https://www.plural.sh/)).

If you're interested, I can keep you in touch about the advancements on this matter, so you can try before anyone else and give me your feedback 😃
